front/src/applications/operationalStudies/components/ManageTrainSchedule/Itinerary/DisplayVias.jsx

```diff
@@ -5,10 +5,11 @@ import { useSelector, useDispatch } from 'react-redux';
 import { deleteVias, permuteVias, updateViaStopTime } from 'reducers/osrdconf';
 import InputSNCF from 'common/BootstrapSNCF/InputSNCF';
 import { useDebounce } from 'utils/helpers';
+import { getConf } from 'reducers/osrdconf/selectors';
 
 function InputStopTime(props) {
   const { index } = props;
-  const osrdconf = useSelector((state) => state.osrdconf);
+  const osrdconf = useSelector(getConf);
```

Comment:
```suggestion
  const vias = useSelector(getVias);
```
In this file, we only use osrdConf.vias, so better use the selector of vias directly

Reply:
yes